app/javascript/components/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react"
+
+
+class Footer extends Component{
+render() {
+    return(
+        <div>
+            <h3>This is a footer</h3>
+        </div>
+        
+    )
+    }
```

Comment:
indentation here needs to corrected to reflect nested logic and aligned closing delimiters